java/kafka/java-kafka-producer.yaml

```diff
@@ -18,18 +18,20 @@ spec:
       - name: java-kafka-producer
         image: quay.io/strimzi-examples/java-kafka-producer:latest
         env:
-          - name: BOOTSTRAP_SERVERS
+          - name: KAFKA_BOOTSTRAP_SERVERS
             value: my-cluster-kafka-bootstrap:9092
-          - name: TOPIC
+          - name: STRIMZI_TOPIC
             value: my-topic
-          - name: DELAY_MS
+          - name: STRIMZI_DELAY_MS
             value: "1000"
-          - name: LOG_LEVEL
+          - name: STRIMZI_LOG_LEVEL
             value: "INFO"
-          - name: MESSAGE_COUNT
+          - name: STRIMZI_MESSAGE_COUNT
             value: "1000000"
-          - name: PRODUCER_ACKS
+          - name: STRIMZI_PRODUCER_ACKS
             value: "all"
```

Comment:
This does not seem to be mentioned anywhere else. Should it be `KAFKA_ACKS`?

Reply:
Yes it should. Changed to KAFKA_ACKS